CondTools/L1Trigger/interface/WriterProxy.h

```diff
@@ -13,6 +14,8 @@
 
 #include <string>
 
+using namespace edm;
```

Comment:
`using namespace` should be avoided in headers, but this seems to be unnecessary as well (at least as far as the header is concerned).